pom.xml

```diff
@@ -102,6 +102,14 @@
         <maven-surefire-plugin.version>2.19.1</maven-surefire-plugin.version>
         <maven-deploy-plugin.version>2.8.2</maven-deploy-plugin.version>
         <handlebars.version>4.0.6</handlebars.version>
+        <pax-exam.version>4.11.0</pax-exam.version>
+        <pax-exam.logging.version>1.10.1</pax-exam.logging.version>
+        <maven-jar-plugin.version>2.6</maven-jar-plugin.version>
+        <felix.maven.version>3.3.0</felix.maven.version>
```

Comment:
Should we already merge this OSGi specific changes? What do you think?

Reply:
Same as above
>🤔  wasn't sur about it since it keeps track of only versions used in the osgi integration, but i after rethinking it it might be better to keep it separated in the osgi branch.